blended-measurement/blended-measurement.tex

```diff
@@ -0,0 +1,685 @@
+sampling step [P7], and are essentially just a performance optimization.
+
+In multi-epoch mode, we use hybrid models (as described in
+Section~\ref{sec:model-selection}) and consider all objects in a blend
+simultaneously, first fitting with an optimizer [P6] and then Monte Carlo
+sampling [P7].  In both cases we will fit to multiple filters simultenously
+(though perhaps not all filters), and only allow the flux to vary between
+filters (i.e. the models will not support variability -- but see also
+\ref{sec:variable-models} for an alternative).  As in the coadd fitting, the
+structural parameters of the galaxy models will be required to be the same in
+each filter.  The most important use case for the Monte Carlo samples [D8] is
+shear estimation for gravitational lensing, but we anticipate it being useful
+for any study of faint objects for which unbiased population statistics are
+more important than precise measurements of individual objects.  The
+optimizer-based fitting is should produce our best astrometric measurements
+for fainter stars, and may yield better galaxy photometry and morphology
+measurements than the deblended coadd fitting.
+
+The simultaneous optimizer-based fit will also be used as templates for
+another round of deblending (as in Section~\ref{sec:models-as-templates}),
+this time producing deblended pixel values for individual visits [D7].  These
+will be used for forced PSF photometry at the per-epoch positions determined
+from the simultaneous multi-epoch fit.  This will populate the forced source
+catalog [D9], which represent our best estimates of the lightcurves of
+faint variable objects.  We use positions from multi-epoch fitting to
+consistently handle stars with significant proper motions, and we perform only
+PSF photometry, as the vast majority of variable objects are indeed point
+sources.
+
+For all multi-epoch measurements, we include models for transient and fast-
+moving objects detected in difference images [D10], as described in
+Section~\ref{sec:time-domain}. These models will have a free flux
+parameter in each epoch, but will have centroids fixed at the position
+determined from detection image(s).
+
+\subsection{Possible Modifications}
+\label{sec:alternatives}
+
+\subsubsection{Likelihood Coadds}
+
+\label{sec:likelihood-coadds}
+
+Likelihood coadds (also known as Kaiser coadds or detection maps) present an
+intriguing but untested alternative to direct coadds.  They represent an
+optimal combination of images in both image quality and S/N (impossible with
+direct coadds), but cannot be interpreted in the same way as traditional
+coadds and single-epoch images, requiring completely new algorithms for all
+operations performed on them.  As a result, making use of likelihood coadds
+may require considerably more human effort, but it could reduce the need for
+multi-epoch processing (but probably not eliminate it). If they prove viable,
```

Comment:
How much investigation do we need to do to determine if they are viable, necessary, or better?  Do we have to build everything and then have a bake-off, or can we find out in a faster way?